logstash-core/spec/logstash/agent_spec.rb

```diff
@@ -373,7 +373,7 @@
       @t.join
     end
 
-    it "resets the metric collector" do
+    xit "resets the metric collector" do
```

Comment:
https://relishapp.com/rspec/rspec-core/v/2-0/docs/pending/pending-examples#temporarily-pending-by-changing-"it"-to-"xit"